component-css/card.css

```diff
@@ -0,0 +1,235 @@
+.active{
+    background-color: rgb(156 148 148 / 15%);
+    backdrop-filter: blur(19px) saturate(100%) contrast(45%) brightness(130%);
+    border-top-left-radius: 1.5rem;
+    border-bottom-left-radius:1.5rem ;
+    color:var(--banner-text-color) !important;
+}
+
+/*                             CONTENT BOX                    */
+
+
+
+
+.content-box .introduction , .card-component-container {
+    font-size: 1.55rem;
+    padding: 1.2rem;
+    width: inherit;
+    margin: 0.8rem;
+    border-radius: 10px;
+    background: #F5F6F7;
+    width: inherit;
+    box-shadow: 0px 1px 5px 2px rgb(0 0 0 / 46%);
+    -webkit-box-shadow: 0px 1px 5px 2px rgb(0 0 0 / 46%);
+    -moz-box-shadow: 0px 1px 5px 2px rgba(0,0,0,0.46);
+}
+
+ .card-container{
+    
+    margin: 1rem 0rem 1rem 0rem;
+    background-color: transparent;
+    width: 100%;
+    background-color: #E7E8E8;
+    border-radius:8px ;
+    box-shadow: 1px 1px 5px 1px rgba(0,0,0,0.46);
+   -webkit-box-shadow: 1px 1px 5px 1px rgba(0,0,0,0.46);
+   -moz-box-shadow: 1px 1px 5px 1px rgba(0,0,0,0.46);
+    width: 100%;
+ 
+}
+
+.content-box .introduction-text{
+    padding-top:1rem ;
+    line-height: 1.3;
+    word-spacing: 2.5px;
+    font-size: 1.3rem ;
+    font-weight: 500;
+ }
+
+.content-box  .iframe-section{
+    padding: 0.9rem 0rem 0.9rem 0rem;
+    width: inherit;
+}
+.content-box  .iframe-section > iframe{
+    max-width: 100%;
+    overflow-x: auto;
+}
+
+
+.content-box .sub-heading-1,.sub-heading-2,.sub-heading-3,.sub-heading-4{
+    max-width: 100%;
+    font-size: 2.2rem;
+    color: var(--main-primary-color);
+    padding-top: 1rem;
+    padding-bottom: 0.5rem;
+}
+
+/*                                  SIMPLE CARD                                  */
+
+.simple-card{
+    
+    padding: 1rem;
+    background-color: #E7E8E8;
+    border-radius:8px ;
+    box-shadow: 1px 1px 5px 1px rgba(0,0,0,0.46);
+   -webkit-box-shadow: 1px 1px 5px 1px rgba(0,0,0,0.46);
+   -moz-box-shadow: 1px 1px 5px 1px rgba(0,0,0,0.46);
+}
+
+/*                                  IMAGE CARD                                     */
+
+.image-card{
+    display: flex;
+    margin: 0px auto;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    width: 100%;
```

Comment:
This should not be this much large in size ,I think you should decrease the size of this card by specifying width and height accordingly.